### PR TITLE
Add preprod and prod to routing

### PR DESCRIPTION
**Pull request by mfcrocker**
# Motivation and Context
We should be doing re-routing for deployments by script to avoid mistakes

# What has changed
Used an existing script that does the same thing

# How to test?
Use this next time we do a preprod deployment

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -24,6 +26,22 @@ elif [ $1 == ${SIT_ENVIRONMENT} ]; then
...
+elif [ $1 == ${PROD_ENVIRONMENT} ]; then
+  ENVIRONMENT=${PROD_ENVIRONMENT}
+  DOMAIN="apps.prod.cf5.onsclofo.uk"
+  MAINTENANCE_APP="ras-maintenance-prod"
+  MAINTENANCE_URL="ras-maintenance-prod"
+  FRONTSTAGE_APP="ras-frontstage-prod"
+  FRONTSTAGE_URL="surveys.ons.gov.uk"
+  TESTING_URL="ras-frontstage-prod-testing"
 else
   echo "Unknown environment, currently script only supports: $DEV_ENVIRONMENT"
```
Comment (insacuri) [inline deploy.sh]: The fail case needs to be updated

**Review by insacuri — CHANGES_REQUESTED** [review]

**Review by insacuri — APPROVED** [review]

File: deploy.sh [hunk 2]
```diff
@@ -7,6 +7,8 @@ fi
 
 DEV_ENVIRONMENT="dev"
 SIT_ENVIRONMENT="sit"
```
Comment (gavinleeedwards) [inline deploy.sh]: The only dev environments that are used are ci and concourse-latest.  But in either case, the maintenance page isn't deployed into those environments so they should be removed, unless there are plans to deploy the maintenance page into them.

File: deploy.sh [hunk 3]
```diff
@@ -24,8 +26,24 @@ elif [ $1 == ${SIT_ENVIRONMENT} ]; then
...
+  ENVIRONMENT=${PROD_ENVIRONMENT}
+  DOMAIN="apps.prod.cf5.onsclofo.uk"
+  MAINTENANCE_APP="ras-maintenance-prod"
+  MAINTENANCE_URL="ras-maintenance-prod"
+  FRONTSTAGE_APP="ras-frontstage-prod"
+  FRONTSTAGE_URL="surveys.ons.gov.uk"
+  TESTING_URL="ras-frontstage-prod-testing"
 else
-  echo "Unknown environment, currently script only supports: $DEV_ENVIRONMENT"
+  echo "Unknown environment, currently script only supports: $DEV_ENVIRONMENT, $SIT_ENVIRONMENT, $PREPROD_ENVIRONMENT and $PROD_ENVIRONMENT"
```
Comment (gavinleeedwards) [inline deploy.sh]: The script doesn't currently support dev environments so it could just be changed to preprod/prod.

File: deploy.sh [hunk 4]
```diff
@@ -24,8 +26,24 @@ elif [ $1 == ${SIT_ENVIRONMENT} ]; then
   FRONTSTAGE_APP="ras-frontstage-sit"
   FRONTSTAGE_URL="ras-frontstage-sit"
   TESTING_URL="ras-frontstage-sit-testing"
+elif [ $1 == ${PREPROD_ENVIRONMENT} ]; then
+  ENVIRONMENT=${PREPROD_ENVIRONMENT}
+  DOMAIN="apps.prod.cf5.onsclofo.uk"
```
Comment (gavinleeedwards) [inline deploy.sh:13]: Should this domain be there or is it a secret?  I'm not sure.

**Review by gavinleeedwards — CHANGES_REQUESTED** [review]

File: deploy.sh (hunk 4 above)
Comment (mfcrocker) [reply deploy.sh:13]: Am checking with TL :)

Comment (necrophonic) [reply deploy.sh:13]: Checked with IA - seems fine to us but we're going to enter things like this onto a page Dan is putting together to ensure we've captured that they've been discussed and agreed

**Review by necrophonic — APPROVED** [review]

**Pull request merged**